src/blake2b.c

```diff
@@ -3,17 +3,25 @@
 #include <stdint.h>
 #include <string.h>
 
+#include "sia.h"
+
 void blake2b_init(cx_blake2b_t *S) {
-    cx_blake2b_init(S, 256);
+    if (cx_blake2b_init_no_throw(S, 256) != CX_OK) {
+        THROW(SW_DEVELOPER_ERR);
+    }
```

Comment:
This doesn't make sense to workaround the deprecated warning of the THROW version by using the no_throw and throwing later.
The purpose of the deprecation was to avoid THROW mechanism which often leads to errors when the exception is catched.
The best would be to cascade properly the error, though as this is not always really necessary as some error shouldn't appear in normal conditions, we are preparing a new ASSERT mechanism which could be use instead. You can follow its progress here: https://github.com/LedgerHQ/ledger-secure-sdk/pull/473